lambda_function.py

```diff
@@ -0,0 +1,88 @@
+import json
+import requests
+import os
+import urllib.parse
+
+
+def lambda_handler(event, context):
+    # Base URL of Gupshup's OTP API, including all constant params
+    baseURL = os.environ.get("OTP_API_BASE_URL")
+    response_headers = {"Access-Control-Allow-Origin": "*"}
+    eventPath = str(event["path"])
+
+    # Check for query params, otherwise return 404 (Parameters not found)
+    if not event["queryStringParameters"]:
+        return {
+            "statusCode": 404,
+            "headers": response_headers,
+            "body": json.dumps("Parameters not found"),
+        }
+
+    queryParams = event["queryStringParameters"]
```

Comment:
```suggestion
    query_params = event["queryStringParameters"]
```
... And so on
make sure you go through all the code and change the variable names so they are in snake_case